js/background.js

```diff
@@ -1,80 +1,76 @@
 /* This script runs in the background from when the app is loaded */
 
-var Domain = "https://channeli.in";
-var Host = "channeli.in";
+var NET_CHECK_DOMAIN = "http://info.cern.ch/";
```

Comment:
If the internet goes down, the intranet still works. So the NET_CHECK_DOMAIN should be a channeli url to keep the extension up and running in such instances